price-estimator/openapi.yml

```diff
@@ -101,6 +104,11 @@ components:
       enum: [baseunits, atoms]
       default: baseunits
       example: baseunits
+    OrderbookKind:
+      type: string
+      enum: [withroundingbuffer, raw]
+      default: withroundingbuffer
+      example: withroundingbuffer
```

Comment:
maybe it's just me but I find `orderbookKind=withroundingbuffer` a bit weird to read. Would maybe addRoundingBuffer=true/false be more intuitive?

Reply:
Comes down to the enum vs bool debate maybe? I don't really have a preference.

Reply:
Yes, it's almost equivalent (in the code I don't mind the verbosity and use of an enum). Just from the perspective of API user friendly-ness I find the above mentioned param a bit ugly. But no strong feelings.

Reply:
I'm OK either way, both `disableRoundingBuffer=true` and `orderbookKind=raw` sound nice to me. However, I think the former would be a little easier to understand when looking at a URL in isolation since the value of `raw` doesn't really capture that the rounding buffer is being disabled, and is the one that you would usually see since `orderbookKind=withroundingbuffer` is the default value.

Reply:
What about `roundingBuffer=enabled` and `disabled`?

Reply:
Love it.